### DISCUSSION TITLE
Alist中如何设置端口为80/443从而实现cf workers加速

### DISCUSSION THREAD

**Discussion by liueic** [AlistGo/alist#5884] [Q&A] [answered]
我用的是一键脚本+宝塔面板的形式，宝塔可以反代到80端口，但是还是用不了加速，nginx会对端口80/443进行占用，是不是将nginx端口改了让给Alist就行

**Comment by sd14** [comment, marked as answer]
cf-worker加速不需要80/443端口，默认5244就行了

**Reply by liueic** [reply]
明白了，可能是我配置里面没有加端口号导致无法访问
